cli/waiter/querying.py

```diff
@@ -0,0 +1,35 @@
+import concurrent
+import logging
+import os
+from concurrent import futures
+
+from waiter import http
+
+
+def query_across_clusters(clusters, query_fn):
+    """Attempts to query entities from the given clusters."""
+    count = 0
+    all_entities = {'clusters': {}}
+    max_workers = os.cpu_count()
+    logging.debug('querying with max workers = %s' % max_workers)
+    with concurrent.futures.ThreadPoolExecutor(max_workers=max_workers) as executor:
+        future_to_cluster = {query_fn(c, executor): c for c in clusters}
+        for future, cluster in future_to_cluster.items():
+            entities = future.result()
+            all_entities['clusters'][cluster['name']] = entities
+            if entities.get('using_pools', False):
+                for pool_entities in entities['pools'].values():
+                    count += pool_entities['count']
+            else:
+                count += entities['count']
+    all_entities['count'] = count
+    return all_entities
+
+
+def get_token(cluster, token_name, include=None):
+    """Gets the token with the given name from the given cluster"""
+    params = {'token': token_name}
+    if include:
+        params['include'] = include
```

Comment:
Does this support multiple include parameters? E.g. if we ever decide to use `include=metadata&include=deleted`?

Reply:
Yes, you can pass a list